helfi_proxy.module

```diff
@@ -82,3 +83,38 @@ function helfi_proxy_page_attachments_alter(array &$attachments) {
     $attachments['#attached']['html_head'][] = [$helfi_content_type, $tag_name];
   }
 }
+
+/**
+ * Implements hook_simple_sitemap_links_alter().
+ */
+function helfi_proxy_simple_sitemap_links_alter(array &$links, $sitemap_variant) {
+  /** @var \Drupal\Core\Config\ImmutableConfig $config */
+  $config = \Drupal::service('config.factory')->get('helfi_proxy.settings');
+
+  if (!$paths = implode("\n", $config->get(ProxyManagerInterface::ROBOTS_PATHS) ?? [])) {
+    return;
+  }
+
+  /** @var \Drupal\helfi_api_base\Environment\Environment $environment */
+  $environment = \Drupal::service('helfi_api_base.environment_resolver')->getActiveEnvironment();
```

Comment:
This can fail to a `\InvalidArgumentException` exception. We should catch it and return void.